File: app/src/main/java/com/example/msdc/api/KeywordResponse.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class KeywordResponse {
    @SerializedName("keywords")
    private final List<KeywordResult> keywords = new ArrayList<>();

    public List<KeywordResult> getKeywords() {
        return keywords;
    }
}

File: app/src/main/java/com/example/msdc/api/KeywordResponse.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class KeywordResponse (
    @field:SerializedName("keywords") val keywords: ArrayList<KeywordResult>? = null
) : Parcelable